wise_banking_api_client/cli.py

```diff
@@ -0,0 +1,148 @@
+"""Command line interface for wise_banking_api_client."""
+
+from typing import Any, Callable, Optional
+
+try:
+    import click
+except ImportError:
+    raise ImportError(
+        "You need to install wisebanking_api_client[cli]:\n\n    pip install wisebanking_api_client[cli]"
+    )
+from pathlib import Path
+
+
+HERE = Path(__file__).parent
+LICENSE = (HERE / "LICENSE").read_text()
+
+
+DEFAULT_PRIVATE_KEY = "wise.com.private.pem"
+DEFAULT_PUBLIC_KEY = "wise.com.public.pem"
+
+
+def opt_license(*param_decls: str, **kwargs: Any) -> Callable:
+    """Show the license
+
+    This is copied from the --help option.
+    """
+
+    def callback(
+        ctx: click.Context, param: click.Parameter, value: bool
+    ) -> None:  # noqa: FBT001, ARG001
+        if not value or ctx.resilient_parsing:
+            return
+        click.echo(LICENSE)
+        ctx.exit()
+
+    if not param_decls:
+        param_decls = ("--license",)
+
+    kwargs.setdefault("is_flag", True)
+    kwargs.setdefault("expose_value", False)
+    kwargs.setdefault("is_eager", True)
+    kwargs.setdefault("help", "Show the license and exit.")
+    kwargs["callback"] = callback
+    return click.option(*param_decls, **kwargs)
+
+
+ARG_API_KEY = click.argument("api_key", envvar="WISE_API_KEY", required=True)
```

Comment:
@niccokunzmann I like the utility of having a CLI interface for various actions; and reading variables from `env` variables seems better than, for example, placing them on the commandline, for various reasons... however: environment variables don't seem ideal for long-lived credentials.  Maybe there are alternatives here?

Reply:
As an alternative, one could use the [Bitwarden secrets manager](https://bitwarden.com/products/secrets-manager/)...

Reply:
What do some other transfer-capable payment client libraries do?